cdap-client-tests/pom.xml

```diff
@@ -63,6 +63,12 @@ the License.
     </dependency>
     <dependency>
       <groupId>co.cask.cdap</groupId>
+      <artifactId>cdap-data-fabric</artifactId>
```

Comment:
I have it spread across multiple commits. I will try to move them in a separate single commit if it is definitely required. Otherwise, I will add enough to the JIRA to make it easy to remove later.